Mods/vcmi/Content/config/english.json

```diff
@@ -138,6 +138,12 @@
 	"vcmi.lobby.deleteFile" : "Do you want to delete following file?",
 	"vcmi.lobby.deleteFolder" : "Do you want to delete following folder?",
 	"vcmi.lobby.deleteMode" : "Switch to delete mode and back",
+	"vcmi.lobby.battleOnlyMode" : "Battle only mode",
+	"vcmi.lobby.battleOnlyModeBattleground" : "Battleground",
+	"vcmi.lobby.battleOnlyModeBattlegroundSelect" : "Select Battleground",
```

Comment:
Should be called "battlefield", not "battleground"

Can be replaced everywhere - e.g. via search-and-replace